src/model/floorplan.ts

```diff
@@ -108,8 +109,8 @@ module BP3D.Model {
      * @param end he end corner.
      * @returns The new wall.
      */
-    public newWall(start: Corner, end: Corner): Wall {
-      var wall = new Wall(start, end);
+    public newWall(start: Corner, end: Corner, height: number = -1): Wall {
```

Comment:
This seems a bit messy. Why is it necessary to be able to create walls with non-default height?